SYCL/SubGroup/shuffle_fp64.cpp

```diff
@@ -16,7 +16,12 @@
 
 int main() {
   queue Queue;
-  check<double>(Queue);
-  std::cout << "Test passed." << std::endl;
+  if (Queue.get_device().has(sycl::aspect::fp64)) {
+    check<double>(Queue);
+    std::cout << "Test passed." << std::endl;
+  } else {
+    std::cout << "Test skipped because device doesn't support aspect::fp64"
+              << std::endl;
+  }
```

Comment:
I'm not sure if I'm going to ask to do that as part of this PR, but the best way would be to merge all shuffle_*.cpp tests into one using this method.